Added advancment post for release v0.7 (#7)
OpenFreeEnergy/alchemiscale.org-site: content/advancements/release-v0.7/index.md

We wanted to take a moment to highlight these releases, and the improvements they bring for users!

## v0.7.0

As a major release, [v0.7.0](https://github.com/OpenFreeEnergy/alchemiscale/releases/tag/v0.7.0) introduced two significant features that advance **alchemiscale**'s capabilities for automated, large-scale execution.

First, this release introduced the **`ComputeManager`** base class, which lays the foundation for upcoming compute autoscaling implementations.
This abstraction enables **alchemiscale** to dynamically scale compute resources across different platforms, including HPC clusters and Kubernetes environments.
With this foundation in place, compute services can now be managed more intelligently, responding to workload demands without manual intervention.
Keep an eye out for [**alchemiscale-k8s**](https://github.com/OpenFreeEnergy/alchemiscale-k8s), which leverages this new capability!

Second, we added the **`Strategist`** service, which enables automated execution of `Strategy` objects for `AlchemicalNetwork`s.
A `Strategy` proposes where to apply additional compute given the set of results currently available for a given `AlchemicalNetwork`.

> Users can now submit a `Strategy` alongside their `AlchemicalNetwork`s, and the `Strategist` service will periodically apply it server-side to adaptively sample `Transformation`s until the `Strategy`'s stop thresholds are satisfied.

Reviewer: "alongside" makes it sounds like it cannot just be assigned at any point to an arbitrary `AlchemicalNetwork`.

```suggestion
Users can now assign a `Strategy` to any of their `AlchemicalNetwork`s, and the `Strategist` service will periodically apply it server-side to adaptively sample `Transformation`s until the `Strategy`'s stop thresholds are satisfied.
```
